repair/repair.cc

```diff
@@ -2218,12 +2255,19 @@ static future<> init_messaging_service_handler(sharded<database>& db, sharded<ne
                 return checksum_range(db, keyspace, cf, range, hv);
             });
         });
+        ms.register_node_ops_cmd([] (const rpc::client_info& cinfo, node_ops_cmd_request req) {
```

Comment:
and here, we'd capture the tracker so we don't need to pick it up from a global (again, later).

Reply:
ok